src/main/java/ch/uzh/ifi/hase/soprafs21/controller/UserController.java

```diff
@@ -60,12 +57,10 @@ public String login(@RequestBody UserPostDTO userPostDTO){
     @ResponseBody
     public void logout(@PathVariable Long userId, @RequestHeader("Auth-Token")String token){
 
-        throw new UnsupportedOperationException("Not implemented yet");
-
         // checks if user id and token are from the same user
         //userService.authorizationCheck(userId, token);
```

Comment:
Will be provided via #67
```suggestion
        userService.isUserAuthenticated(userId, token);
```